root/etc/e-smith/templates/etc/ejabberd/ejabberd.yml/30MacrosTLS20181001

```diff
@@ -4,14 +4,22 @@ my $tlsPolicy = $tls{'policy'} || '0';
 
 if ($tlsPolicy >= 20181001) {
     $OUT .=q(
+#
+# TLS policy 20181001
+#
+
 define_macro:
   'CERTFILE': "/etc/ejabberd/ejabberd.pem"
   'TLSOPTS':
     - "no_sslv3"
     - "no_tlsv1"
     - "cipher_server_preference"
-  'CIPHERS':
-    - "ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-SHA384:ECDHE-ECDSA-AES128-SHA256:EDH+CAMELLIA:EDH+aRSA:EECDH+aRSA+AESGCM:EECDH+aRSA+SHA384:EECDH+aRSA+SHA256:EECDH:+CAMELLIA256:+AES256:+CAMELLIA128:+AES128:+SSLv3:!aNULL:!eNULL:!LOW:!3DES:!MD5:!EXP:!PSK:!DSS:!RC4:!SEED:CAMELLIA256-SHA:AES256-SHA:CAMELLIA128-SHA:AES128-SHA"
+  'CIPHERS': "ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-ECDSA-CHACHA20-POLY1305:
+    ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-SHA384:
```

Comment:
Newlines seems to be accepted in the cipher suite specification string, as they do not lead to different cipher lists.